docs/source/configuration/overview.mdx

```diff
@@ -109,77 +103,71 @@ without downtime
 </td>
 <td>
 
-Prints out the Apollo Router's version
+Prints out the Apollo Router's version.
 
 </td>
 </tr>
 
 </tbody>
 </table>
 
-## Configuration-awareness in your text editor
-
-The Apollo Router is able to produce a JSON schema for use in your text editor. It significantly eases the burden of making
-sure that your yaml file is correctly formatted, and even gives you content assist.
-
-To generate the schema use:
-```bash
-router --schema > configuration_schema.json
-```
-
-One you have generated the schema configure your text editor. Here are some instructions for well known editors:
-* [Emacs](https://emacs-lsp.github.io/lsp-mode/page/lsp-yaml)
-* [IntelliJ](https://www.jetbrains.com/help/idea/json.html#ws_json_using_schemas)
-* [Sublime](https://github.com/sublimelsp/LSP-yaml)
-* [Vim](https://github.com/Quramy/vison)
-* [VsCode](https://code.visualstudio.com/docs/languages/json#_json-schemas-and-settings)
-
 ## Configuration file
 
-Apollo Router takes an optional YAML configuration file as input via the `--config` option. All supported configuration options are shown below:
+The Apollo Router takes an optional YAML configuration file as input via the `--config` option.
 
 ### Listen address
 
-By default, the Apollo Router starts a HTTP server listening on `127.0.0.1:4000`. This can be
-changed in the configuration file:
+By default, the router starts an HTTP server that listens on `127.0.0.1:4000`. You can specify a different address like so:
 
 ```yaml title="router.yaml"
 #
 # server: Configuration of the HTTP server
 #
 server:
   # The socket address and port to listen on
-  # Defaults to 127.0.0.1:4000
-  listen: 127.0.0.1:4000
+  # (Defaults to 127.0.0.1:4000)
+  listen: 127.0.0.1:4001
```

Comment:
Hopefully not too nitty of me, but I'm going to suggest `5000` for this illustration since `4001` is frequently in use by a subgraph.

```suggestion
  listen: 127.0.0.1:5000
```

I'd claim that `5000` will just be less likely a surprise to _someone_.